lib/galaxy/model/mapping.py

```diff
@@ -123,6 +123,22 @@
     Column('lifetime', Integer),
     Column('assoc_type', VARCHAR(64)))
 
+model.OIDCAccessToken.table = Table(
```

Comment:
Since in addition to access token, we have id and refresh token among other values, I guess it would be better to have a more generic class name; e.g., `OIDCToken`.

Reply:
Similarly, I guess table name can be more generic; e.g., `oidc_token`.

Reply:
Sounds good 👍